recipes-kernel/linux/files/bcm2712-raspberrypi5-xen.dts

```diff
@@ -29,6 +29,24 @@
 			xen,enhanced = "enabled";
 		};
 	};
+
+	/* Tell Xen about OP-TEE */
+	reserved-memory {
```

Comment:
Hm.  Seems a bit entangled - may be we can re-use DT provided by FW? In boot script copy it where we need instead of loading from disk. (Yep static changes in dts will be lost)